### PR TITLE
Release/v0.0.3

### PR DESCRIPTION
**Pull request by SabineHaas**
Fix #Issue

**Changes proposed in this pull request**:
- Bump version number
- Hot fix: correct parameter `overwrite_grid_parameters` in `run_pvcompare.py`

The following steps were realized, as well (required):
- [ ] Update the CHANGELOG.md
- [ ] Check if full simulation tests pass locally (`EXECUTE_TESTS_ON=master pytest`)

Also the following steps were realized (if applies):
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [ ] Write test(s) for your new patch of code

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

In case of an error due to linting, run `black . --exclude docs/` and push your changes.
Note that in case you do not fix a whole issue you should start your PR with `Address #xyz`.

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by SabineHaas** [comment]
I have installed pvcompare on this branch in a clean virtual environment. `EXECUTE_TESTS_ON=master pytest` runs fine
`78 passed, 63 warnings in 309.28s (0:05:09) `

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Here is a template for new release sections
 -
 ```
 
-## [unreleased]
+## [0.0.3] - 2021-04-21
```
Comment (MaGering) [inline CHANGELOG.md:22]: Isn't it `[0.0.3] - 2021-04-22`?

**Review by MaGering — DISMISSED** [review]
Thanks for the hotfix in commit https://github.com/greco-project/pvcompare/pull/294/commits/342079870b761d1c2b20e2a4dd47a9a13d27d103! :)

File: CHANGELOG.md (hunk 1 above)
Comment (SabineHaas) [reply CHANGELOG.md:22]: oh ohhhh living in the past.. you are right! I'll fix it right now

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,9 +19,38 @@ Here is a template for new release sections
 -
 ```
 
+## [0.0.3] - 2021-04-21
```
Comment (SabineHaas) [inline CHANGELOG.md:22]:
```suggestion
## [0.0.3] - 2021-04-22
```

**Comment by SabineHaas** [comment]
I won't wait for the checks, as I've only change the date in the changelog and it is docmented here that for 3420798 the build passes.

**Pull request merged**